cypress/integration/users/create-users.spec.js

```diff
@@ -0,0 +1,21 @@
+describe('Creating user', () => {
+  beforeEach(() => {
+    cy.login('diku_admin', 'admin');
+  });
+
+  it('should be possible', function () {
+    Cypress.on('uncaught:exception', () => {
+      return false;
+    });
+
+    cy.visit('/users/create');
+    cy.get('#adduser_lastname').type('Test123');
```

Comment:
please use interactors instead

additionally wrap it in commands, e.g. fillUserInfo - more reusable pieces of code